### ISSUE TITLE
(Tinkerer's) Silk Touch pickaxe on Hive breaks hive, no drops

### ISSUE THREAD

**Issue by rswhisper** [ForestryMC/ForestryMC#1670]
forestry_1.10.2-5.2.17.368
TConstruct-1.10.2-2.6.3

Silky Touch Iron Pickaxe, made with Tinker's Construct, broke a Wintry Hive.
Nothing dropped at all.

Intentional or bug?

**Comment by MauveCloud** [comment]
Probably intentional - it's the wrong tool type (hives are supposed to be broken with scoops, not pickaxes), so silk touch won't help.  Similar to how, in vanilla Minecraft, a silk touch diamond shovel won't work to harvest a diamond ore block.

Also, I tried starting from creative mode to give a Forestry scoop silk touch (which is allowed using an anvil and an enchanted book, I didn't bother checking if the scoop can be directly enchanted with an enchanting table), but it dropped bees instead of the hive, indicating that hives are not subject to silk touch.

**Comment by temp1011** [comment]
Should silk touch be allowed to break hives? Personally I don't think it's a good idea. But the current behaviour is intentional.

**Comment by bookerthegeek** [comment]
I don't see any reason why silk touching a hive would break anything, so why not?

**Comment by temp1011** [comment]
For decoration purposes I think it's better to have some kind of decorative hive block. Collecting hives with a pickaxe just seems a bit silly to me and I guess also makes collecting many bees easier without needing backpacks etc. Not sure about quarries with silk touch collecting hives too (if they do already that seems a bit silly). I think it could make sense for the scoop to work with silk touch but the scoop has such a low durability that there would probably need to be a new tier introduced to make it worthwhile (proven scoop?). I also don't actually know how enchanting the scoop works at the moment.

However, there's not necessarily a clear answer. This is just my opinion :smile: .

**Comment by MauveCloud** [comment]
FWIW, GregTech scoops can have considerably more uses than the default Forestry scoops, depending on material, and there are a few materials that would give silk touch.

**Comment by temp1011** [comment]
True, I still think forestry should have a better one too though for the late game. I think something like this is best considered in 1.13 because to check for the item held and whether it has silk touch without nasty code this will require https://github.com/MinecraftForge/MinecraftForge/pull/4727.

**Comment by bookerthegeek** [comment]
Use the same durability as a normal scoop. Just add unbreaking and mending .
